samples/sampler/stories/default/DayPicker.js

```diff
@@ -15,13 +15,15 @@ export const _DayPicker = (args) => (
 		<DayPicker
 			aria-label={args['aria-label']}
 			disabled={args['disabled']}
+			selected={args['selected']}
 			onSelect={action('onSelect')}
 		/>
 	</Scroller>
 );
 
 text('aria-label', _DayPicker, DayPicker);
 boolean('disabled', _DayPicker, DayPicker);
+number('selected', _DayPicker, DayPicker);
```

Comment:
`selected` can be either a number or an array. So this type seems wrong.
And normally this prop isn't set by a user. So I prefer not to have this prop in the default sampler but it seems okay for qa-sampler.